### PR TITLE
add Travis CI

### PR DESCRIPTION
**Pull request by kellyselden**
to test #207

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: node_js
```
Comment (cvan) [inline .travis.yml]: can you remove the Windows line-feed return?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+language: node_js
+node_js:
+  - "node"
+  - "6"
+  - "4"
+
+cache:
+  directories:
+    - node_modules
```
Comment (cvan) [inline .travis.yml]: same here

**Comment by cvan** [comment]
this otherwise looks good to me. r? @toji

**Comment by cvan** [comment]
@toji, @HalfdanJ: when you get a chance, could you review this PR?

**Comment by cvan** [comment]
this otherwise looks fine to me. unless you have any other feedback, @toji, you can merge this.

File: .travis.yml (hunk 1 above)
Comment (kellyselden) [inline .travis.yml]: fixed.

**Pull request merged**